include/aos/iam/identhandler.hpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_IDENTHANDLER_HPP_
+#define AOS_IDENTHANDLER_HPP_
+
+#include "aos/iam/config.hpp"
+
+namespace aos {
+namespace iam {
+namespace identhandler {
+
+/** @addtogroup iam Identification and Access Manager
+ *  @{
+ */
+
+/**
+ * Ident observer interface.
+ */
+class IdentObserverItf {
+public:
+    /**
+     * Update observer.
+     *
+     * @param[in] updateMessage update message
+     * @returns Error.
+     */
+    virtual Error Update(const String& updateMessage) = 0;
+
+    /**
+     * Destroys ident observer interface.
+     */
+    virtual ~IdentObserverItf() = default;
+};
+
+/**
+ * Ident handler interface.
+ */
+class IdentHandlerItf {
+public:
+    /**
+     * Returns System ID.
+     *
+     * @param[out] systemdID result system ID.
+     * @returns Error.
+     */
+    virtual Error GetSystemID(String& systemdID) = 0;
+
+    /**
+     * Returns unit model.
+     *
+     * @param[out] unitModel result unit model.
+     * @returns Error.
+     */
+    virtual Error GetUnitModel(String& unitModel) = 0;
+
+    /**
+     * Returns subjects.
+     *
+     * @param[out] subjects result subjects.
+     * @returns Error.
+     */
+    virtual Error GetSubjects(Array<String>& subjects) = 0;
```

Comment:
`Should be Array<StaticString<cSubjectLen>>` . `cSubjectLen` should be defined in common/types.hpp